rules.neon

```diff
@@ -33,3 +33,7 @@ services:
 			reportUnknownTypes: %doctrine.reportUnknownTypes%
 		tags:
 			- phpstan.rules.rule
+
+conditionalTags:
+	PHPStan\Rules\Doctrine\ORM\EntityRule:
```

Comment:
Please rename the rule to something like `FinalEntityClassRule` and also add it to `services` (without any tags).

I also guess this is the reason why the build failed, so please fix that.